Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/ruleset/RulesetManagement.java

```diff
@@ -48,6 +51,11 @@ public class RulesetManagement implements RulesetManagementInterface {
      */
     private static final Logger logger = LogManager.getLogger(RulesetManagement.class);
 
+    /**
+     * Language used internally in places where language does not matter.
```

Comment:
Could you clarify this comment? "...in places where language does not matter" - so where _does_ it matter?

Reply:
comment extended